### PR TITLE
AND <-> OR transcription

### PR DESCRIPTION
**Pull request by wlammen**
Now start with the ordenary AND - OR mixed op stuff.  First rules to transscribe one to the other.
I think the tests are skipped, because I already checked in commit [a2d4cc2](https://github.com/metamath/set.mm/pull/2750/commits/a2d4cc2fd5e45066fb680ab988926b62e7c39277) in PR #2748

### CODE REVIEW

**Review by benjub — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -7742,13 +7742,6 @@ inference eliminates the need for a syllogism ( ~ syl ) -type inference
       -> ( ( ( ph \/ ps ) \/ ch ) -> ( ph \/ ch ) ) ) $=
     ( wi wo orel2 ax-1 ja orim1d ) BADABEZACBAJADBAFAJGHI $.
 
-$(
--.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-  Assorted results not assigned yet
--.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-
-$)
-
```
Comment (avekens) [inline set.mm:7751]: There should be a section header for AND - OR mixed theorems, shouldn't it?

**Review by avekens — CHANGES_REQUESTED** [review]

File: set.mm (hunk 1 above)
Comment (wlammen) [reply set.mm:7751]: Yes.  @benjub wanted to improve on the one  of the previous section, so I wait until he is ready. See PR #2739.

Perhaps you can agree with postponing this to a later PR, and keep an eye on it, so the work is not blocked by something not under my control.

Just to fix this issue, I added a quick and dirty text, marked with a TODO.  That is the best that I can do for the moment.

Comment (benjub) [reply set.mm:7751]: Yes, let's do the sectioning and titles after the reordering is done. Not much left to do, as I understand, @wlammen ?

Comment (wlammen) [reply set.mm:7751]: Not really.  This is on the level of janitorial work.

Comment (benjub) [reply set.mm:7751]: Opened Issue #2751 for that.

File: set.mm [hunk 2]
```diff
@@ -7636,6 +7636,24 @@ inference eliminates the need for a syllogism ( ~ syl ) -type inference
   Mixed conjunction and disjunction
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
+  TODO: The comment to the previous section has issues that need to be
+  resolved.  In particular the notion of duality is deemed inappropriate in the
+  context of Metamath.  So here is just a small introductionary note, that
```
Comment (benjub) [inline set.mm]: Can you remove that sentence about duality ?
The rest is OK to me, and I'll try to propose uniform section comments building on it and the previous ones soon.

Comment (wlammen) [reply set.mm]: done

**Pull request merged**